### PR TITLE
Make cli.action mockable

### PR DESCRIPTION
**Pull request by markpundsack**
Add cli.writeError and make cli.action use it.

### CODE REVIEW

File: lib/console.js [hunk 1]
```diff
@@ -60,5 +73,6 @@ function debug (obj) {
 exports.hush  = hush;
 exports.log   = log;
```
Comment (dickeyxxx) [inline lib/console.js]: can we add `writeLog` alongside this?

**Comment by markpundsack** [comment]
@dickeyxxx Added writeLog too. I'm unsure why console.error needs .apply when used in this context. I left it out in process.stderr.write. It seems to work. :)

**Comment by markpundsack** [comment]
I still haven't validate this works. My tests on heroku-pipelines are still failing.

**Comment by dickeyxxx** [comment]
use npm link first then https://docs.npmjs.com/cli/link

**Pull request merged**